src/sandbox/jakub/login/schema.ts

```diff
@@ -1,8 +1,14 @@
-import * as y from 'yup'
+import * as zod from 'zod'
 
-export const LoginFormSchema = y.object({
-  email: y.string().email().required(),
-  password: y.string().required(),
+export const LoginFormSchema = zod.object({
+  email: zod
+    .string()
+    .nonempty('This is required')
+    .email({ message: 'Must be a valid email' }),
+  password: zod
+    .string()
+    .nonempty('This is required')
+    .min(8, { message: 'Too short - min. 8 characters' }),
```

Comment:
1. Domyślnie `zod` ustawia wszystkie pola jako `required`, `nonempty` odnosi się do typu `array`, którego tutaj nie mamy 
2. Domyślnie `zod` ustawia `required_error` na "Required", jeżeli chcesz zmienić to na "This is required" musisz do stringa wrzucić {required_error: "...."}. Możemy zostawić "Required" lub zmienić na "This is required", mi obojętnie, wybierz co wolisz.

```suggestion
  export const LoginFormSchema = z.object({
  email: z
    .string({ required_error: 'This is required' })
    .email({ message: 'Must be a valid email' }),
  password: z
    .string({ required_error: 'This is required' })
    .min(8, { message: 'Too short - min. 8 characters' }),
})
```